hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/SCMCertificateClient.java

```diff
@@ -109,6 +109,9 @@ protected InitResponse handleCase(InitCase init)
       } else {
         return FAILURE;
       }
+    case EXPIRED_CERT:
+      LOG.error("SCM CA certificate is about to be expire!");
```

Comment:
Better use warn instead of error here since SCM could still function well in a period of time.